homeassistant/components/teslemetry/strings.json

```diff
@@ -274,5 +286,10 @@
         "name": "State code"
       }
     }
+  },
+  "exceptions": {
+    "no_cable": {
+      "message": "Unable to lock as no charge cable is connected"
```

Comment:
Could this say something as "Locking is automatic when charge cable is connected"?